dtmi/azureiot/phone_as_a_device/phoneasadevice-1.json

```diff
@@ -0,0 +1,175 @@
+{
+    "@id": "dtmi:azureiot:Phone_As_A_Device:PhoneAsADevice;1",
+    "@type": "Interface",
+    "contents": [
+        {
+            "@type": "Component",
+            "name": "accelerometer",
+            "schema": "dtmi:azureiot:Phone_As_A_Device:Sensors:Accelerometer;1"
+        },
+        {
+            "@type": "Component",
+            "name": "battery",
+            "schema": "dtmi:azureiot:Phone_As_A_Device:Sensors:Battery;1"
+        },
+        {
+            "@type": "Component",
+            "name": "location",
+            "schema": "dtmi:azureiot:Phone_As_A_Device:Sensors:Location;1"
+        },
+        {
+            "@type": "Component",
+            "name": "magnet",
+            "schema": "dtmi:azureiot:Phone_As_A_Device:Sensors:Magnet;1"
+        },
+        {
+            "@type": "Component",
+            "name": "pressure",
+            "schema": "dtmi:azureiot:Phone_As_A_Device:Sensors:Pressure;1"
+        },
+        {
+            "@type": "Component",
+            "displayName": {
+                "en": "Device information"
```

Comment:
I would avoid to include localized elements

Reply:
The localization occurs in the previous version of these templates https://github.com/Azure/iot-plugandplay-models/blob/main/dtmi/azureiot/phoneasadevice-1.json and localization is part of the DTDL specification (https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/dtdlv2.md#localization-examples).